src/tests/JIT/opt/Remainder/IntRemainder.cs

```diff
@@ -11,18 +11,6 @@ static class IntRemainder
         static int _fieldValue = 123;
         static uint _fieldValueUnsigned = 123;
 
-        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
why are there changes in this file?

Reply:
> But won't the disassembly still have the helper call (the fallback)?

the test expects exact order, helper -> div. it may or may not happen (we migth disable this opt for JIT in favor of NAOT only) so to avoid changing it back and forth I just deleted it since it doesn't look like it has a big value

Reply:
Got it.

Reply:
@TIHan What were you trying to test here?
- That xor/wzr are used?
- That the static constructor call isn't omitted?
- That the ordering was preserved?

As commented here, the last one isn't a requirement for the code.  I think using `X64-DAG` for both would handle this, but I'm not sure what your intention was here.